### PR TITLE
Favicon and more

### PR DESCRIPTION
**Pull request by ac-jorellanaf**
```
Added favicon. Fixed layout issues with different screen sizes.
Added dataset list headers. Fixed applied filter badges overflowing.
Allowed for sidebar to be partly rendered for when no datasets are added.
Added warning that no datasets were found. Fixed behaviour of GHGA logo link.
- Closes GDEV-680
- Closes GDEV-685
- Fixes GDEV-683
- Fixes GDEV-691
- Fixes GDEV-706
```

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -17,4 +17,4 @@
   <div id="root"></div>
```
Comment (alamgirmani) [inline public/index.html:17]: the favicon.png file belongs to the assets folder. Create an images subfolder and move the file to images folder.

File: src/components/browse/datasetlist/datasetListHeader.tsx [hunk 2]
```diff
@@ -17,22 +17,31 @@ const DatasetListHeader = (props: dataSetListHeaderProps) => {
     ) {
       let searchParamsList = props.searchParams.get("f").split(";");
       for (var item of searchParamsList) {
-        filterParamsList.push(item.replace(":", " | "));
+        filterParamsList.push(item.replace(":", ": "));
```
Comment (alamgirmani) [inline src/components/browse/datasetlist/datasetListHeader.tsx:20]: why is this changed?

File: src/components/browse/sidebar/filter.tsx [hunk 3]
```diff
@@ -39,30 +41,33 @@ const Filter = (props: filterProps) => {
...
-                    <Col lg md sm xl xs xxl={2} className="h-100">
-                      <Form.Label className="p-0 m-0" htmlFor={key}>
-                        {option.count}
-                      </Form.Label>
-                    </Col>
-                  </Row>
-                </Form.Label>
-              </div>
-            );
+            if (option.option.includes(",") !== true) {
```
Comment (alamgirmani) [inline src/components/browse/sidebar/filter.tsx:44]: The response from the backend has 11 facets for the "type" and in the UI we are now just showing 7 facets. I dont think we can ignore the facets that has ","

**Review by alamgirmani — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (ac-jorellanaf) [reply public/index.html:17]: No. It must be in the public folder. The home page cannot access data from the src folder.

File: src/components/browse/datasetlist/datasetListHeader.tsx (hunk 2 above)
Comment (ac-jorellanaf) [reply src/components/browse/datasetlist/datasetListHeader.tsx:20]: It took slightly less space.

File: src/components/browse/sidebar/filter.tsx (hunk 3 above)
Comment (ac-jorellanaf) [reply src/components/browse/sidebar/filter.tsx:44]: Yes, because all the facets that had a comma were facets that were there because the Dataset had multiple study types. Deepak said he was going to remove returning these at some point because they are completely redundant. If you look at dev, you will notice that all facets that have a comma are already present as facets without commas.

Comment (alamgirmani) [reply src/components/browse/sidebar/filter.tsx:44]: Yes, but this is a fix that needs to come from the backend. We dont have to do it. We display the facets as we receive.

File: src/components/browse/datasetlist/datasetListHeader.tsx (hunk 2 above)
Comment (alamgirmani) [reply src/components/browse/datasetlist/datasetListHeader.tsx:20]: This is a cosmetic feature, I think a "pipe" is lot of visible than the "colon". We can check with the design team if they have any preference and update it.

File: src/components/browse/sidebar/filter.tsx (hunk 3 above)
Comment (ac-jorellanaf) [reply src/components/browse/sidebar/filter.tsx:44]: Ok

**Review by alamgirmani — APPROVED** [review]
